.github/workflows/test-sim-merge.yml

```diff
@@ -71,39 +71,6 @@ jobs:
           ENGINE_PORT: 8551
           ETH_PORT: 8661
 
-      - name: Pull geth withdrawals
```

Comment:
looks like we need to remove the whole file?

Reply:
Not yet, one other PR in progress then we can remove this file and a lot of utility code along it.